pika/adapters/twisted_connection.py

```diff
@@ -84,89 +94,172 @@ class TwistedChannel(object):
                        'tx_rollback', 'flow', 'basic_cancel')
 
     def __init__(self, channel):
-        self.__channel = channel
-        self.__closed = None
-        self.__calls = set()
-        self.__consumers = {}
+        self._channel = channel
+        self._closed = None
+        self._calls = set()
+        self._consumers = {}
 
         channel.add_on_close_callback(self.channel_closed)
 
     def channel_closed(self, _channel, reason):
```

Comment:
`channel_closed()` appears to be for internal use, so should be named with a leading underscore as a private method

Reply:
Right, I'll fix this.